lib/operations/tables.js

```diff
@@ -305,9 +305,15 @@ export function alterColumn(tableName, columnName, options) {
   const columnsStr = formatLines(actions, `  ALTER "${columnName}" `);
   const columnCommentStr =
     typeof columnComment !== "undefined"
-      ? `\n${comment("TABLE ", columnName, columnComment)}`
+      ? `\n${comment("COLUMN ", `${tableName}"."${columnName}`, columnComment)}`
```

Comment:
this was a bug which producess `COMMENT ON TABLE ${columnName} ...` instead of `COMMENT ON COLUMN ${tableName}.${columnName} ...`
maybe you want to give a schema, tableName and columnName to your comment function